climakitae/core/data_export.py

```diff
@@ -26,6 +27,37 @@
 bytes_per_gigabyte = 1024 * 1024 * 1024
 
 
+def remove_zarr(dir_path):
+    try:
+        shutil.rmtree(dir_path)
+        print(f"Zarr dataset '{dir_path}' deleted successfully.")
+    except FileNotFoundError:
+        print(f"Zarr dataset '{dir_path}' not found.")
+    except OSError as e:
+        print(f"Error deleting Zarr dataset '{dir_path}': {e}")
+
+
+def _add_metadata(data):
```

Comment:
Is `_add_metadata` replacing `_update_attributes`?

Reply:
`_update_attributes` was only needed for exporting `netCDF` to S3, so it is gone. `_add_metadata` is the `DataSet` level metadata that was being written to the original dataset (and therefore was still there after export). I have moved that to this function that is run on the copy of the data so that doesn't happen any more.